src/Foundatio.Aliyun/Storage/AliyunFileStorage.cs

```diff
@@ -113,8 +138,14 @@ public class AliyunFileStorage : IFileStorage {
             }
             oldpath = NormalizePath(oldpath);
             newpath = NormalizePath(newpath);
-            return await CopyFileAsync(oldpath, newpath, cancellationToken).AnyContext() &&
-                   await DeleteFileAsync(oldpath, cancellationToken).AnyContext();
+            try {
+                return await CopyFileAsync(oldpath, newpath, cancellationToken).AnyContext() &&
+                       await DeleteFileAsync(oldpath, cancellationToken).AnyContext();
+            }
+            catch (Exception ex) {
+                ExceptionDispatchInfo.Capture(UnwrapException(ex)).Throw();

```

Comment:
I think this is capture sets the stack trace to the inner exception and rethrows it. But it's probably better to just not even catch it and let the end user handle the exception (not wroth the extra overhead)